internal/cypress/config.go

```diff
@@ -135,3 +146,11 @@ func Validate(p Project) error {
 
 	return nil
 }
+
+// standardizeVersionFormat remove the leading v in version to ensure reliable comparisons.
+func standardizeVersionFormat(version string) string {
```

Comment:
I think this could be a public method in the general config package?

Reply:
Yes, since it will be used across different frameworks.